src/main/java/CustomListException.java

```diff
@@ -0,0 +1,5 @@
+public class CustomListException extends Exception{
```

Comment:
CustomListException - the name of the exception should state what went wrong - in the logs, you will see a big log, and then the class name - so it should mean, what exactly went wrong. Also read, what is the difference between a check and uncheck exception, when to use which one.